projects/aws/eks-a-admin-image/provisioners/al2/download_eksa_artifacts.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
does this file need to be specific for al2? I would expect this kind of eks-a commands to be OS agnostic